chapter06/drjt/ex1.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $storage = 'coconet.archive';
+
+my %total_bytes;
+if ( -r $storage ) {
```

Comment:
You specifically check the that the file is readable here (and in exercise 2), but you don't specifically check that the file is writeable before writing your output to it.